src/main/java/net/sf/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -72,7 +72,7 @@ private void readJournalList(Reader in) {
     }
 
     private void addLine(String line) {
-        if (line.startsWith("#")) {
+        if (!line.isEmpty() && (line.charAt(0) == '#')) {
```

Comment:
It may not be better... PMD claims that line.charAt(0) is "better" than startsWith, but in this case the additional line.isEmpty may cancel the benefit.